db/migrate/20161012202458_create_atlases.rb

```diff
@@ -0,0 +1,9 @@
+class CreateAtlases < ActiveRecord::Migration
+  def change
+    create_table :atlases do |t|
+      t.references :user
```

Comment:
You need to add indices for all of these relationships, so in this case, `add_index :user_id`, otherwise every query involving these columns is going to be a sequential table scan (i.e. slower than it needs to be). Much more important for the potentially larger tables like `atlas_alterations`